Formula/glassfish.rb

```diff
@@ -1,8 +1,8 @@
 class Glassfish < Formula
   desc "Open Source Jakarta EE Platform Implementation"
   homepage "https://eclipse-ee4j.github.io/glassfish/"
-  url "https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip&mirror_id=518"i
```

Comment:
I dont think you can make that change, it missed the bin folder which is needed to boot up an application server for java apps.
![image](https://user-images.githubusercontent.com/1580956/57588931-31e87d00-74ea-11e9-8742-d051b33ed550.png)